aws-private-existing-vpc/module/main.tf

```diff
@@ -0,0 +1,186 @@
+data "aws_vpc" "selected" {
+  default = "false"
+  state   = "available"
+}
+
+data "aws_subnets" "subnets" {
+  filter {
+    name   = "vpc-id"
+    values = [data.aws_vpc.selected.id]
+  }
+  filter {
+    name   = "tag:Name"
+    values = ["*${var.subnet_name}*"] 
+  }
+}
+
+
+resource "aws_ecs_cluster" "aws-ecs-cluster" {
+  name = "${var.app_name}-${var.app_environment}-cluster"
+
+  setting {
+    name  = "containerInsights"
+    value = "enabled"
+  }
+
+  tags = {
+    Name        = "${var.app_name}-ecs"
+    Environment = var.app_environment
+  }
+}
+
+resource "aws_cloudwatch_log_group" "log-group" {
+  name = "${var.app_name}-${var.app_environment}-logs"
+
+  tags = {
+    Application = var.app_name
+    Environment = var.app_environment
+  }
+}
+
+
+locals {
```

Comment:
I saw you removed the use of "env_vars.json" file. Do you have any particular reason for that?

Reply:
I did. But none I'm attached to. Adding it back.